dev/breeze/src/airflow_breeze/commands/release_management_commands.py

```diff
@@ -815,7 +815,10 @@ def publish_docs(
 
 @release_management.command(
     name="add-back-references",
-    help="Command to add back references for documentation to make it backward compatible",
+    help="Command to add back references for documentation to make it backward compatible. "
+    "By default it generates back references for all providers. You can also specify "
+    "'apache-airflow' or 'helm-chart` as parameter to generate back references for them or you "
+    "can specify short ids of packages explicitly (including 'apache-airflow' and 'helm-chart').",
```

Comment:
We know that by default it generates for providers but we should also include that in the description here

Reply:
It's not default any more - I will shorten the description now. It's not needed any more - options explain everything,